influunt-api/app/models/Evento.java

```diff
@@ -38,17 +39,27 @@
 
     @Column
     @NotNull(message = "não pode ficar em branco")
-    private String numero;
+    private Integer posicao;
 
     @Column
     @Enumerated(EnumType.STRING)
-    @NotNull(message = "não pode ficar em branco")
```

Comment:
Dia da semana pode ser `null`? Se a data não for `null`, dá para inferir qual dia da semana é o evento.